src/Messages.jl

```diff
@@ -140,19 +141,19 @@ HTTP.Response(200, headers; body = "Hello")
 Response() = Request().response
 
 Response(s::Int, body::AbstractVector{UInt8}) = Response(s; body=body)
-Response(s::Int, body::AbstractString) = Response(s, bytes(body))
+Response(s::Int, body::AbstractString) = Response(s; body=bytes(body))
```

Comment:
I'm pretty sure this and the method just below were just wrong by not passing `body` as a keyword argument